FirebaseAuth/Sources/Public/FirebaseAuth/FIRAuthErrors.h

```diff
@@ -418,9 +422,41 @@ typedef NS_ERROR_ENUM(FIRAuthErrorDomain, FIRAuthErrorCode){
      */
     FIRAuthErrorCodeBlockingCloudFunctionError = 17105,
 
-    /** Indicates an error for when the client identifier is missing.
+    /** Indicates that the recaptcha integration is not enabled for this project.
+     */
+    FIRAuthErrorCodeRecaptchaNotEnabled = 17200,
+
+    /** Indicates that the recaptcha token is missing when sending request to the backend.
+     */
+    FIRAuthErrorCodeMissingRecaptchaToken = 17201,
+
+    /** Indicates that the recaptcha token is invalid when sending request to the backend.
+     */
+    FIRAuthErrorCodeInvalidRecaptchaToken = 17202,
+
+    /** Indicates that The recaptcha action is invalid when sending request to the backend.
+     */
+    FIRAuthErrorCodeInvalidRecaptchaAction = 17203,
+
+    /** Indicates an error for when the client type is missing.
+     */
+    FIRAuthErrorCodeMissingClientType = 17204,
+
+    /** Indicates that the recaptcha version sent to backend is missing.
+     */
+    FIRAuthErrorCodeMissingRecaptchaVersion = 17205,
+
+    /** Indicates that the recaptcha version sent to backend is invalid.
+     */
+    FIRAuthErrorCodeInvalidRecaptchaVersion = 17206,
+
+    /** Indicates that the request type sent to backend is invalid.
+     */
+    FIRAuthErrorCodeInvalidReqType = 17207,
+
+    /** Indicates that the recaptcha sdk is not linked.
      */
-    FIRAuthErrorCodeMissingClientIdentifier = 17993,
+    FIRAuthErrorCodeRecaptchaSDKNotLinked = 17208,
```

Comment:
Is this thrown by the backend or SDK only? We should document this in the spreadsheet of error numbers

Reply:
It's SDK only. Added to the doc.